redaxo/src/addons/phpmailer/README.md

```diff
@@ -7,6 +7,7 @@
     - [Spam-Blocker](#spam-blocker)
     - [Verwendung bei selbstsignierten Zertifikaten](#zertifikate)
     - [Verschlüsselung: Automatische TLS-Verbindung](#autotls)
+    - [Senden über multiple Domains](#multipledomains)
```

Comment:
Ich finde hier das „multiple“ unpassend. Vllt besser in deutsch statt denglisch?

Reply:
nun, multiple ist weder deutsch noch englisch ;-)